### PR TITLE
Update README with a KV Store example

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CRTXDOC-1796

## Description
A request was made to provide an example of how to create a KV Store.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SplunkPy/Integrations/SplunkPy/README.md [hunk 1]
```diff
@@ -247,6 +247,48 @@ The following features are not supported in non-ES (Enterprise Security) Splunk.
...
+
+1. In Cortex XSOAR, create a new KV Store.
+   
+   **!splunk-kv-store-collection-create kv_store_name="\<kv_store_name\>"**
+   
+   For example:
+   
+   **!splunk-kv-store-collection-create kv_store_name=”test_kvstore”**
+   
+3. Define the fields and their type in the KV Store.
```
Comment (ilappe) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:261]: the numbers aren't consistent
should be here 2

File: Packs/SplunkPy/Integrations/SplunkPy/README.md [hunk 2]
```diff
@@ -247,6 +247,48 @@ The following features are not supported in non-ES (Enterprise Security) Splunk.
...
+
+   **!splunk-kv-store-collection-config kv_store_collection_name="\<kv_store_name\>" kv_store_fields="field.\<field-name\>=\<type\>,index.\<index-name\>=\<type\>,field.\<field-name-or-index\>=\<type\>,..."**
+
+   For example:
+   
+   **!splunk-kv-store-collection-config kv_store_collection_name="test_kvstore" kv_stre_fields="field.src=cidr,field.t=number,field.description=string"**
+
+   **Note:** To see the fields in Splunk, you must install the _Splunk App for Lookup File Editing_ app in Splunk.
```
Comment (ilappe) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:269]: can we put here link to Splunk documentation?

**Review by ilappe — APPROVED** [review]
Great!

File: Packs/SplunkPy/Integrations/SplunkPy/README.md (hunk 1 above)
Comment (ShirleyDenkberg) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:261]:
```suggestion
2. Define the fields and their type in the KV Store.
```

File: Packs/SplunkPy/Integrations/SplunkPy/README.md [hunk 3]
```diff
@@ -247,6 +247,48 @@ The following features are not supported in non-ES (Enterprise Security) Splunk.
...
+
+4. Make the KV Store usable in Splunk queries.
```
Comment (ShirleyDenkberg) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:271]:
```suggestion
3. Make the KV Store usable in Splunk queries.
```

File: Packs/SplunkPy/Integrations/SplunkPy/README.md (hunk 2 above)
Comment (ShirleyDenkberg) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:269]:
```suggestion
   **Note:** To see the fields in Splunk, you must install the _Splunk App for Lookup File Editing_ app in Splunk. For more information, see [Define a KV Store lookup in Splunk](https://docs.splunk.com/Documentation/Splunk/9.1.0/Knowledge/DefineaKVStorelookupinSplunkWeb#:~:text=Splunk%20Web%20currently,Editing%20on%20Splunkbase).
```

Comment (ShirleyDenkberg) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:269]:
```suggestion
   **Note:** To see the fields in Splunk, you must install the _Splunk App for Lookup File Editing_ app in Splunk. For more information, see see [Define a KV Store lookup in Splunk](https://docs.splunk.com/Documentation/Splunk/9.1.0/Knowledge/DefineaKVStorelookupinSplunkWeb#:~:text=Splunk%20Web%20currently,Editing%20on%20Splunkbase).
```

Comment (ShirleyDenkberg) [reply Packs/SplunkPy/Integrations/SplunkPy/README.md:269]:
```suggestion
   **Note:** To see the fields in Splunk, you must install the _Splunk App for Lookup File Editing_ app in Splunk. For more information, see [Define a KV Store lookup in Splunk](https://docs.splunk.com/Documentation/Splunk/9.1.0/Knowledge/DefineaKVStorelookupinSplunkWeb#:~:text=Splunk%20Web%20currently,Editing%20on%20Splunkbase).
```

**Pull request merged**